src/mgmt/tests/mocks.c

```diff
@@ -443,10 +443,9 @@ mock_ds_mgmt_tgt_update_setup(void)
 int     ds_mgmt_pool_extend_return;
 uuid_t  ds_mgmt_pool_extend_uuid;
 int
-ds_mgmt_pool_extend(uuid_t pool_uuid, d_rank_list_t *svc_ranks,
-		    d_rank_list_t *rank_list,
-		    char *tgt_dev,  size_t scm_size, size_t nvme_size,
-		    size_t domains_nr, uint32_t *domains)
+ds_mgmt_pool_extend(uuid_t pool_uuid, d_rank_list_t *svc_ranks, d_rank_list_t *rank_list,
+		    char *tgt_dev, size_t scm_size, size_t nvme_size, size_t domains_nr,
+		    uint32_t *domains, size_t qlc_size)
```

Comment:
```suggestion
ds_mgmt_pool_extend(uuid_t pool_uuid, d_rank_list_t *svc_ranks, d_rank_list_t *rank_list,
		    char *tgt_dev, size_t scm_size, size_t nvme_size, size_t nvme_bulk_size, size_t domains_nr,
		    uint32_t *domains)
```

As far as I am concern, all the `*_size` arguments should be next to each other.